### PR TITLE
feat(tokens): Add an tokens screen

### PR DESCRIPTION
**Pull request by pmuir**

### CODE REVIEW

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: 8a4e4c6ae73f353c257edbc0d8e7b3054df37c5b
  * Your commit message does not contain a body

Guidelines are available at https://github.com/fabric8io/fabric8-ui

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: 1104b01de71fa1c4544f8456b3f6a5aa69a40514
  * Your commit message does not contain a body

Guidelines are available at https://github.com/fabric8io/fabric8-ui

---
This message was auto-generated by https://gitcop.com

File: package.json [hunk 1]
```diff
@@ -188,7 +188,7 @@
     "rollup-plugin-angular-aot": "0.0.4",
     "rollup": "0.41.4",
     "run-sequence": "1.2.2",
-    "sass-loader": "6.0.2",
+    "sass-loader": "6.0.3",
```
Comment (joshuawilson) [inline package.json]: At this point all the projects are in sync on library version numbers. If you need this update, then we should apply it to all the other libraries we are working with.

Comment (joshuawilson) [inline package.json]: If anything is out of sync, it is unintentional and should be fixed.

**Comment by pmuir** [comment]
6.0.3 means we can have empty scss files again. I will raise PRS for all
repos. Some already moved eg. Runtime console.

**Comment by pmuir** [comment]
All other repos are on 6.0.3

File: package.json (hunk 1 above)
Comment (pmuir) [inline package.json]: All other repos are on 6.0.3 already. fabric8-ui had got out of sync.

**Comment by joshuawilson** [comment]
Then that is on me for not updating this one. Thanks.

**Pull request merged**